src/ixdat/readers/zilien.py

```diff
@@ -35,10 +35,16 @@ def read(self, path_to_file, cls=None, name=None, **kwargs):
 
         from EC_MS import Zilien_Dataset
 
-        if cls is Measurement:
-            cls = ECMSMeasurement
-
-        if "technique" not in kwargs:
+        if "technique" in kwargs:
```

Comment:
Nice. I'm actually re-writing the Zilien parser at this moment, but I will make sure to include this part.